backend/tests/Feature/Models/EntryModelTest.php

```
@@ -0,0 +1,146 @@
+            $model->countIncrease();
+            $model->refresh();
+
+            $updatedAtAfterIncrease = $model->updated_at;
+            if ($updatedAtAfterIncrease === null) {
+                $this->fail('updated_at was not set after refresh.');
+            }
+
+            $this->assertSame(1, $model->count);
+            $this->assertTrue($updatedAtAfterIncrease->eq($originalUpdatedAt));
+        }
+    }
```

PHPMD (2.15.0)

[error] The method test_count_increase_increments_count_without_touching_updated_at is not named in camelCase. (undefined)

(CamelCaseMethodName)